File: src/main/java/org/chousik/handlers/CommandHandler.java
package org.chousik.handlers;

import org.chousik.commands.ACommand;

import java.util.HashMap;

/**
 * Класс управления списком команд
 */
public class CommandHandler {
    private HashMap<String, ACommand> commands = new HashMap<>();

    /**
     * Добавление команды в список
     *
     * @param commandName - имя команды
     * @param Command     - команда
     */
    public void addCommand(String commandName, ACommand Command) {
        commands.put(commandName, Command);
    }

    /**
     * Получение списка команд
     *
     * @return список команд
     */
    public HashMap<String, ACommand> getCommands() {
        return commands;
    }
}

File: src/main/kotlin/handlers/CommandHandler.kt
package org.chousik.handlers

import org.chousik.commands.ACommand

class CommandHandler(private val commands: HashMap<String, ACommand>) {

    fun addCommand(commandName: String, aCommand: ACommand) {
        commands[commandName] = aCommand
    }

    fun getCommands(): HashMap<String, ACommand> {
        return commands
    }
}